swtbot_test/src/utilities/Utility.java

```diff
@@ -436,8 +436,6 @@ public int compare(Action o1, Action o2) {
 						Utility.changeBoard(model, null, null, true, false);
 					}
 				}
-			} else if (action.getActionId().equalsIgnoreCase("buildAll")) {
```

Comment:
Q: Why need to remove this?

Reply:
I put buildAll function in line 154 of TCExecute.java, since if we put build all in testcase.xml, it will build before finish checking for configuration error, therefore the time for build will be recorded wrongly, since this tc is only 1 line, I suggest we don't handle it through testcase.xml